.github/workflows/testing_this_now.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
weird choice for a file name :smile_cat:

Reply:
haha just so I can run it quickly on the CI... will delete this before it's merged.